.changeset/new-pugs-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"evervault-python": major
+---
+
+Migrated Function run requests to new API. Added more comprehensive error handling for Function runs. Deprecated encryptAndRun, listing Functions, async Function run requests, and specifying version of Function to run.
```

Comment:
I would be slightly more specific here and explain how error handling was updated. The changelog should inform users on how their integration could be impacted for breaking changes!

Reply:
I'll update this once we have the errors finalized in the subsequent PR 👍